README.md

```diff
@@ -99,6 +99,18 @@ You can see the live demo at: https://linkshub.vercel.app
    pnpm run dev
    ```
 
+> **P.S**: In case you have `docker` installed in your system, you can follow these steps to setup the environment:
+> 1. After forking and cloning the repo(as mentioned above), get into the project directory:
+> ```bash
+> cd LinksHub/
+> ```
+> 2. Start the docker container with:
+> ```bash 
+> docker compose up
+> ```
+> 3. Now start adding your changes.
+> 4. No need to restart the container again and again after starting it once, because the changes you make will reflect into the container instantly.
```

Comment:
[Nitpick] This seems to provide additional advice, so it sounds better as a note instead of step.
```suggestion
> **Note:** You don't need to restart the container again and again after starting it once, because the changes you make will reflect into the container instantly.
```